R/response-status.r

```diff
@@ -46,11 +46,12 @@ status_code.numeric <- function(x) x
 http_status <- function(x) {
   status <- status_code(x)
 
-  status_desc <- http_statuses[[as.character(status)]]
-  if (is.na(status_desc)) {
+  if (isFALSE(status %in% names(http_statuses))) {
```

Comment:
`isFALSE()` isn't a good approach here because it's only available in recent versions of R, and it's strictness isn't needed as we know that `%in%` always returns a logical vector with the same length as the LHS.

Reply:
`isFALSE` was here mostly to handle cases if status is `NA` for some reason. But apparently, `%in%` already handles it as well. Force changed the code.